test/cli/whole-program_test.py

```diff
@@ -358,3 +358,18 @@ def test_checkclass_project_builddir_j(tmpdir):
     build_dir = os.path.join(tmpdir, 'b1')
     os.mkdir(build_dir)
     __test_checkclass_project(tmpdir, ['-j2', '--cppcheck-build-dir={}'.format(build_dir)])
+
+
+def test_ctu_fail(tmpdir):
```

Comment:
This file only contain tests which relate to the `whole-program` folder so this should be added the `other_test.py`.